celery/backends/couchdb.py

```diff
@@ -82,7 +82,7 @@ def get(self, key):
             return None
 
     def set(self, key, value):
-        data = {'_id': key, 'value': value}
+        data = {'_id': key.decode('ascii'), 'value': value}
```

Comment:
In order to maintain Python2/3 compatibility, I believe it is generally advisable to use the compatibility functions in [five](https://github.com/celery/celery/blob/master/celery/five.py) (which in its turn uses [vine.five](https://github.com/celery/vine/blob/master/vine/five.py)).

So, for consistency perhaps it would be better to use `string`:

```
from celery.five import string

data = {'_id': string(key), 'value': value}
```